src/main/java/io/github/pixee/security/HtmlEncoder.java

```diff
@@ -21,4 +19,292 @@ private HtmlEncoder() {}
   public static String encode(final String s) {
     return Escape.html(s);
   }
+
```

Comment:
I can't leave a comment on it, but the `createSafeObjectInputStream()` from this type I assumed would be in the Java 8 version of this class for the JAR, so that we could use it. This seems like it will be only available in Java 11.

I see a couple solutions, but it seems like we should create a new type, `SandboxingObjectInputStream` that contains just that particular method, and leave this file as is. That way, everyone can access `SandboxingObjectInputStream`, and the `ObjectInputFilter`-related will only be in the 11 binary.

Reply:
Oh, I hadn't noticed that, i saw the class name on `ObjectInputFilters` and just assumed the whole class was a a java 8 thing. I'll fix with the new type